kornia/geometry/quaternion.py

```diff
@@ -314,6 +312,17 @@ def from_axis_angle(cls, axis_angle: Tensor) -> "Quaternion":
         """
         return cls(axis_angle_to_quaternion(axis_angle))
 
+    def to_axis_angle(self) -> Tensor:
+        """Convert the quaternion to a rotation matrix of shape :math:`(B, 3, 3)`.
```

Comment:
Docs are wrong

Reply:
Fixed (note that I did not specify the output shape - Quaternion does not enforce the data to have a batch dimension) even if the docs are written as it did. The conversion works both with and without batch dimension. But Quaternion.identity().matrix() gives an output of shape (1, 3, 3) even if Quaternion.data is of shape (4, ).
This inconsistency in both the conversion functions and the documentation is worth fixing.

Reply:
I see, thanks to look at it. Are you open to fix that ?

Reply:
Yes, I will create an issue, where we can discuss and take a look at it.